utils/tte.js

```diff
@@ -15,22 +15,24 @@ const ZERO = 0;
 
 /**
  * Create TTE
+ * @param {String} id - pager duty id
  * @param {String} team - team paged
  * @param {Array} pdData - list of pagerduty data
  * @returns {Object} - A tte
  */
-function createTTE(team, pdData) {
+function createTTE(id, team, pdData) {
   const tte = {};
-  tte.startTime = pdData.body.log_entries
+  tte.start = pdData.body.log_entries
```

Comment:
is the ttx bot using start now or startTime

Reply:
Yes